Formula/g/glslang.rb

```diff
@@ -26,6 +26,10 @@ class Glslang < Formula
   depends_on "spirv-tools"
   uses_from_macos "python" => :build
 
+  # Fix SPIRV-Tools-opt dependency which breaks cmake files used by other projects
+  # Slightly modified version of https://github.com/KhronosGroup/glslang/pull/3487, noted in comments there
```

Comment:
When can it be removed?

Reply:
This can be removed when upstream fixes the -config.cmake files to either add the find_dependency() for SPIRV-Tools-opt, or adjusts the directive so that linking against a shared libglslang does not add `-lSPIRV-Tools-opt` to the linker flags.

This is tracked in the referenced upstream PR. The current PR is a partial fix; if they only merge the partial fix, I will open another PR.

Reply:
The upstream PR was updated to include a complete fix. When that PR is merged and a release is made, this local patch can be removed.

Reply:
@SMillerDev the patch has been merged upstream and will be in the next release of glslang. How should it be handled here?

Reply:
If we can use the upstream patch, that would be better.

Reply:
This has been fixed to use the upstream patch.

Reply:
Can you use https://github.com/KhronosGroup/glslang/commit/82e0d00b32d2245efd9da8196f139bc0564765d1.patch?full_index=1 then, instead of copying the upstream patch?

Reply:
Done, I thought I did so earlier but apparently didn't commit the change properly.

Reply:
@SMillerDev  the CI failure does not appear related to the PR. Can you rerun the CI?